src/main/io/serial_msp.c

```diff
@@ -587,6 +589,49 @@ static uint32_t packFlightModeFlags(void)
     return junk;
 }
 
+
+typedef struct pgToMSPMapEntry_s {
+    pgn_t pgn;
+    uint8_t mspId;
+    uint8_t mspIdForSet;
+} pgToMSPMapEntry_t;
+
+static const pgToMSPMapEntry_t pgToMSPMap[] =
+{
+    { PG_BOARD_ALIGNMENT, MSP_BOARD_ALIGNMENT, MSP_SET_BOARD_ALIGNMENT },
+    { PG_FAILSAFE_CONFIG, MSP_FAILSAFE_CONFIG, MSP_SET_FAILSAFE_CONFIG },
+};
+
+#define PG_TO_MSP_MAP_ENTRY_COUNT (sizeof(pgToMSPMap) / sizeof(pgToMSPMapEntry_t))
+
+uint8_t pgMatcherForMSPSet(const pgRegistry_t *candidate, const void *criteria)
+{
+    uint8_t index;
+    uint8_t mspIdForSet = *(uint8_t *)criteria;
```

Comment:
doing that generates a warning though...

```
    const pgRegistry_t *reg = pgMatcher(pgMatcherForMSP, (void*)cmdMSP);
```

`../main/io/serial_msp.c:609:27: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]`

suggestions?